app/src/main/java/org/fossasia/openevent/events/NoInternetEvent.java

```diff
@@ -0,0 +1,7 @@
+package org.fossasia.openevent.events;
+
+/**
+ * Created by MananWason on 8/4/2015.
+ */
+public class NoInternetEvent {
```

Comment:
I mean this was my thought behind doing this. There is no assumption. We can put good log messages and show the message as "trouble downloading data" or something like this. Thoughts?